packages/amplify-cli/src/lib/init-steps/s0-analyzeProject.js

```diff
@@ -123,7 +123,7 @@ async function getEnvName(context) {
   const isEnvNameValid = (inputEnvName) => {
     let valid = true;
 
-    if (inputEnvName.length > 10 || inputEnvName.length < 2 || /[^a-z]/g.test(inputEnvName)) {
+    if (inputEnvName.length > 11 || inputEnvName.length < 2 || /[^a-z]/g.test(inputEnvName)) {
```

Comment:
Thanks for the PR, but we have validation text in the code which says that "Environment name should be between 2 and 10 charecters", like in line 149. You might want to make corresponding changes out there as well.